doc/guides/collaborator-guide.md

```diff
@@ -151,30 +151,28 @@ the TSC meeting agenda.
 
 ### Waiting for Approvals
 
-Before landing pull requests, allow 48 hours for input from other Collaborators.
-Certain types of pull requests can be fast-tracked and may land after a shorter
-delay. For example:
+Before landing pull requests, allow for input from other Collaborators.
+If you expect that a pull request might raise objections, it is a good idea to
+wait longer before landing. You can also add the `wait-for-feedback` label
+to pull requests; in that case, it is expected that the pull request will not
+be merged until at least 48 hours after it has been opened, in order to allow
+for everybody to comment on it.
 
-* Focused changes that affect only documentation and/or the test suite:
-  * `code-and-learn` tasks often fall into this category.
-  * `good-first-issue` pull requests may also be suitable.
-* Changes that fix regressions:
-  * Regressions that break the workflow (red CI or broken compilation).
-  * Regressions that happen right before a release, or reported soon after.
+For pull requests that are lacking an approval from a member of the relevant
+team (e.g. @nodejs/http in the case of HTTP changes), it is recommended to keep
+them open for a longer period of time, e.g. several days.
 
-To propose fast-tracking a pull request, apply the `fast-track` label. Then add
-a comment that Collaborators may upvote.
+### Reverting PRs shortly after landing
 
-If someone disagrees with the fast-tracking request, remove the label. Do not
-fast-track the pull request in that case.
+If it has been less than 48 hours after a pull request has been opened and it
+has already been landed, and you object to it in a way that cannot be addressed
+through a new pull request which addresses your concerns, you can open a
+pull request containing a revert and merge it without waiting for approvals or
+CI runs.
```

Comment:
Add link to "Reverting a commit" so folks know what are the right steps to revert a commit.

```suggestion
[pull request containing a revert](#reverting-commits) and merge it without
waiting for approvals or CI runs.
```

Reply:
We should also make sure ncu can recognize revert commits and consider those as valid (otherwise we won't be able to use the commit queue to land).